website/orders/tests/test_services.py

```diff
@@ -0,0 +1,113 @@
+import datetime
+from datetime import timedelta
+from unittest.mock import patch
+
+from django.contrib.auth import get_user_model
+from django.test import TestCase, override_settings
+from django.utils import timezone
+from guardian.shortcuts import assign_perm
+
+from orders import services
+from orders import models
+from venues.models import Venue
+
+
+User = get_user_model()
+
+
+class OrderServicesTests(TestCase):
+
+    fixtures = ["users.json", "venues.json"]
+
+    def setUp(self):
+        venue_pk_1 = Venue.objects.get(pk=1)
+        self.order_venue = models.OrderVenue.objects.create(venue=venue_pk_1)
+        self.shift = models.Shift.objects.create(
+            venue=self.order_venue, start_date=timezone.now(), end_date=timezone.now() + timedelta(hours=4)
+        )
+        self.product = models.Product.objects.create(name="Test product", current_price=1.25)
+        self.product.available_at.add(self.order_venue)
+        self.product.save()
+        self.normal_user = User.objects.get(pk=2)
+
+    @patch("orders.services.logging")
```

Comment:
why?

Reply:
The same as above, not patching this will create log messages in the console.

Reply:
I see... but won't Django automatically fix this in its test framework if we correctly register the loggers in the Django settings? https://docs.djangoproject.com/en/4.0/topics/logging/#topic-logging-parts-loggers